website/docs/d/schema_template_vrf_contract.html.markdown

```diff
@@ -26,17 +26,10 @@ data "mso_schema_template_vrf_contract" "demovrf01" {
 
 ## Argument Reference ##
 
-
-* `schema_id` - (Required) The schema-id where vrf is associated.
-* `template_name` - (Required) template associated with the vrf.
-* `vrf_name` - (Required) name of the vrf with contract to be attached.
-* `relationship_type` - (Required) Type of relation between VRF and Contract. Allowed values are `provider` and `consumer`.
-* `contract_name` - (Required) Name of contract to be attached with the VRF.
-
-
-
-## Attribute Reference ##
-* `contract_schema_id` - (Optional) SchemaId of contract. 
-* `contract_template_name` - (Optional) Name of template where contract is residing.
-
-
+* `schema_id` - (Required) The schema ID of the VRF.
+* `template_name` - (Required) The template name of the VRF.
+* `vrf_name` - (Required) The name of the VRF.
+* `relationship_type` - (Required) The relationship type of the VRF with Contract. Allowed values are `provider` and `consumer`.
+* `contract_name` - (Required) The name of the Contract.
+* `contract_schema_id` - (Optional) The schema ID of the Contract. The `schema_id` of the VRF will be used if not provided. 
+* `contract_template_name` - (Optional) The template name of the Contract. The `contract_template_name` of the VRF will be used if not provided. 
```

Comment:
should it be Optional -> Read-only ?

Reply:
no since this is possible input on datasource